source/material_model/grain_size.cc

```diff
@@ -1345,6 +1396,14 @@ namespace aspect
                  ExcMessage("The parameter 'Use paleowattmeter' has been removed. "
                             "Use the parameter 'Grain size evolution formulation instead'."));
 
+          volume_fraction_phase_one             = Utilities::string_to_double
+                                                  (Utilities::split_string_list(prm.get ("Volume fraction of phase")));
```

Comment:
This is just a single double, not a list of doubles, right?

Reply:
Yes, thank you!